DevGuide.md

```diff
@@ -81,11 +82,162 @@ Every type has an entry inside the `types` dictionary in [`pokete_data/types.py`
+                "x": 20,
+                "y": 0,
+                "txt": CENTER
+            },
+            "house1": {
+                "x": 2,
+                "y": 0,
+                "txt": HOUSE1
+            },
+            "shop": {
+                "x": 29,
+                "y": 3,
+                "txt": SHOP
+            },
+```
+You can also add you own buildings if you want like this:
+```Python
+"hotel": {
+                "x": 43,
+                "y": 18,
+                "txt": r"""    ..___________
+                              /||          /\
+                             / ||         /  \
+                            /___Hotel!___/    \
+                            | ##      ## | ## |
+                            |    ____    |    |
+                            |____|  |____|____|"""
+            },
+```
+You should also add terrain such as:
+```Python
+"hard_obs": {
+            "tree_group_1": {
+                "x": 35,
+                "y": 2,
+                "txt": """ (()(()((())((()((()
+())(())))())))()))(()
+ || ||| ||||| |||||"""
+            },
+    }
+```
+You can also add soft obstacles where you can find poketes such as:
+```Python
+"soft_obs": {
+            "meadow1": {
+                "x": 1,
+                "y": 8,
+                "txt": """     ;;;;;;;
+                            ;;;;;;;;;;;;
+                         ;;;;;;;;;;;;;;;;
+                        ;;;;;;;;;;;;;;;;;;;
+                         ;;;;;;;;;;;;;;;;;
+                           ;;;;;;;;;;;;;
+                              ;;;;;;    """
+            },
+    }
+```
+If you want to add a door to a building you do that with the `dors:` portion of the dict such as:
+```Python
+"dors": {
+            "dor_playmap_51_1": {
+                "x": 90,
+                "y": 12,
+                "args": {"map": "playmap_51", "x": 1, "y": 28}
+            },
+        }
+```
+If you're adding doors to buildings you should also include a map of the inside of the building such as:
+```Python
+"playmap_22": {
+        "hard_obs": {
+            "inner_walls": {
+                "x": 2,
+                "y": 1,
+                "txt": """________________________
+                          |______________________|
+                          |          a           |
+                          |       ________       |
+                          |                      |
+                          |                      | |
+                          |                        |
+                          |                      __|
+                          |__________  __________|
+                          |_________|  |_________|"""
+            },
+        },
+}
+```
+To add balls to pickup throughout the map you add them like this:
+```Python
+"balls": {
+            "ball_1": {
+                "x": 1,
+                "y": 1
+            },
+            "ball_2": {
+                "x": 7,
+                "y": 12
+            },
+        }
+```
```

Comment:
Maybe it's also important to mention the needed additions to `pokete_data/mapstations.py` to ensure the correct display on the minimap.